eval_metrics.py

```diff
@@ -59,6 +69,10 @@ def main(model_name: str, dataset_name: str):
     results = eval_model(model, dataset)
     print(f"metrics for {model.name}: ", results)
 
+    run["parameters"] = {"model_name": model_name}
```

Comment:
please use model.name instead of  model_name

Reply:
Ok